src/nvim/tui/input.c

```diff
@@ -213,9 +213,7 @@ static void forward_simple_utf8(TermInput *input, TermKeyKey *key)
   while (*ptr) {
     if (*ptr == '<') {
       int n = (size_t)snprintf(buf + len, sizeof(buf) - len, "<lt>");
-      if (n < 0 || n >= sizeof(buf) - len) {
-        break;
-      }
+      assert(n < 0 || n >= sizeof(buf) - len);
```

Comment:
```suggestion
      assert(n >= 0 && n < sizeof(buf) - len);
```

Reply:
Sorry, did this in a rush and completely ignored what the assertion was.